client/src/components/PageHeader/PageHeader.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
```

Comment:
I would rename the file to just `<Header/>` instead of `<PageHeader/>` since it'll convey the same information and keeps it simple.

Reply:
The component name should have the appropriate changes to be Header instead (If you decide to change the file name)